pkg/runtime/application_config_provider.go

```diff
@@ -90,6 +91,47 @@ var externalEventsConfig = config.MustRegisterSection(externalEvents, &interface
 type ApplicationConfigurationProvider struct{}
 
 func (p *ApplicationConfigurationProvider) GetDbConfig() *interfaces.DbConfig {
+	// The error only occurs when using single binary to launch flyte cluster.
```

Comment:
Should this entire package be moved somewhere common? Stdlib or something? Having two packages register the same config section isn't a good pattern... 

Maybe just the db connection part?

Reply:
yeah, agree, I think we should move to `flytestdlib`.